wp-content/plugins/core/functions/templates.php

```diff
@@ -14,3 +17,16 @@ function tribe_template_part( $slug, $name = null, $args = [] ) {
 		return ob_get_clean();
 	}
 }
+
+if ( ! function_exists( 'defer_template_part' ) ) {
+	/**
+	 * @param string      $slug
+	 * @param string|null $name
+	 * @param array       $args
+	 *
+	 * @return Deferred_Component
+	 */
+	function defer_template_part( $slug, $name = null, $args = [] ) {
```

Comment:
Ryan and I had a discussion about naming this function, but there might be better options. Suggestions?

Reply:
Agreed, I'm not sure the name aligns with the use quite right. Maybe something like `nest_template_part()`?

Reply:
I prefer `defer` or maybe stash` to `nest` as this can be utilized in other contexts than just the container component.

Reply:
Hmm... I'm not sure I follow that. In what context would the output from this not be _nested_ within another component?

Reply:
I'm fine with defer.